documentation/manual/src/main/asciidoc/en-US/modules/configuration.asciidoc

```diff
@@ -36,7 +36,7 @@ The provider name is [classname]`org.hibernate.ogm.jpa.HibernateOgmPersistence`.
             <property name="hibernate.transaction.jta.platform"
                       value="JBossTS" />
             <property name="hibernate.ogm.datastore.provider"
-                      value="infinispan" />
+                      value="infinispan_embedded" />
```

Comment:
One was left on purpose, to test the previous name. But you're right, I missed some more.. fixing. Thanks!